### PR TITLE
Add suggestions url field

### LINKED ISSUE #513: Prediction/Suggestions in omnibox for custom search engines/Google
### Problem description

Even with the suggestion/autocomplete setting enabled ("Use a prediction service to help complete searches and URLs typed in the address bar") there are still no suggestions/autocomplete for custom search engine URLs _(it appears to only work for DuckDuckGo)_. 

When adding the Google search string present in chrome (`{google:baseURL}search?q=%s&{google:RLZ}{google:acceptedSuggestion}{google:originalQueryForSuggestion}{google:assistedQueryStats}{google:searchFieldtrialParameter}{google:searchClient}{google:sourceId}{google:instantExtendedEnabledParameter}ie={inputEncoding}`) it doesn't get processed as it does in Chrome, nor does it work by default, an issue which I suspect is closely related to this.

### PR DESCRIPTION
**Pull request by qvint**
This adds the ability to set suggestions url from the UI. Closes #513

add-suggestions-url-field-2

### CODE REVIEW

File: patches/ungoogled-chromium/add-suggestions-url-field.patch [hunk 1]
```diff
@@ -0,0 +1,455 @@
+--- a/chrome/browser/resources/settings/search_engines_page/search_engine_dialog.html
```
Comment (Eloston) [inline patches/ungoogled-chromium/add-suggestions-url-field.patch]: Could you add a brief comment to the top of the patch?

**Comment by Eloston** [comment]
Nice work! One question: Does `suggest_url` from `components/search_engines/prepopulated_engines.json` get filled into the Suggestions URL box for built-in search engines?

**Comment by qvint** [comment]
> Does `suggest_url` from `components/search_engines/prepopulated_engines.json` get filled into the Suggestions URL box for built-in search engines?

Yes, [it does](https://user-images.githubusercontent.com/16843577/48790656-ed311580-ed00-11e8-9c6d-6406a1b62322.png).

**Comment by Eloston** [comment]
Nice. Everything looks good.

**Pull request merged**